app/lib/capital_subtotals.rb

```diff
@@ -1,8 +1,10 @@
 class CapitalSubtotals
-  def initialize(applicant_capital_subtotals: PersonCapitalSubtotals.new, partner_capital_subtotals: PersonCapitalSubtotals.new)
+  def initialize(applicant_capital_subtotals: PersonCapitalSubtotals.new, partner_capital_subtotals: PersonCapitalSubtotals.new, capital_contribution: nil, combined_assessed_capital: nil)
```

Comment:
do we need all these default constructors?

Reply:
We definitely don't need to default `applicant_capital_subtotals` because this should never be instantiated without one of those. I'm less sure about `partner_capital_subtotals` here: sometimes there is no partner, so there are no partner subtotals. We could represent that with setting `partner_capital_subtotals` to nil, but then we have to build in a bunch of nil checks throughout our code. I don't have a strong sense of the best approach here.